### PR TITLE
passed bus driving test

### PR DESCRIPTION
**Pull request by XSnelliusX**
Whats your oppinion on this?
#42

### CODE REVIEW

**Review by MichaelMGraf — APPROVED** [review]
Driving Test definitely passed!

File: what-drives-us/what-drives-us.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# What drives us - Autonomy, Mastery, Purpose
+
+## **Introduktion**
+
+The mental model "What drives us is based on the book written by Daniel Pink. It argues that human motivation is largely intrinsic, and that the aspects of this motivation can be divided into autonomy, mastery, and purpose.
```
Comment (jens1o) [inline what-drives-us/what-drives-us.md:5]:
```suggestion
The mental model "What drives us is based" on the book written by Daniel Pink. It argues that human motivation is largely intrinsic, and that the aspects of this motivation can be divided into autonomy, mastery, and purpose.
```

File: what-drives-us/what-drives-us.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+![wer das liest ist doof](what-drives-us-cartoon.jpg)
```
Comment (jens1o) [inline what-drives-us/what-drives-us.md:7]: ok I love you too honey <3

please change this :)

File: what-drives-us/what-drives-us.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## **Autonomy**
+
+Autonomy  is the need to direct your own life and work. To be fully motivated, you must be able to control what you do, when you do it, and who you do it with.
```
Comment (jens1o) [inline what-drives-us/what-drives-us.md:11]:
```suggestion
Autonomy is the need to direct your own life and work. To be fully motivated, you must be able to control what you do, when you do it, and who you do it with.
```

double space here

File: what-drives-us/what-drives-us.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Autonomy motivates us to think creatively  without needing to conform to strict workplace rules. By rethinking traditional ideas of control (regular office hours, dress codes, ...) organizations can increase staff autonomy, build trust, and improve creativity.
```
Comment (jens1o) [inline what-drives-us/what-drives-us.md:13]:
```suggestion
Autonomy motivates us to think creatively without needing to conform to strict workplace rules. By rethinking traditional ideas of control (regular office hours, dress codes, ...) organizations can increase staff autonomy, build trust, and improve creativity.
```

File: what-drives-us/what-drives-us.md (hunk 2 above)
Comment (Claiyc) [inline what-drives-us/what-drives-us.md:7]: You may want to change this to something nicer...-

**Review by jens1o — CHANGES_REQUESTED** [review]
I really love the way you wrote it. Attached you find a few suggestions and pointers.

File: what-drives-us/what-drives-us.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## **Mastery**
+
+Mastery is the desire to improve. If you're motivated by mastery, you'll likely see your potential as being unlimited, and you'll constantly seek to improve your skills through learning and practice. Someone who seeks mastery needs to attain it for its own sake.
```
Comment (Claiyc) [inline what-drives-us/what-drives-us.md:17]: "You will" instead of "you'll"

File: what-drives-us/what-drives-us.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+## **Purpose**
+
+People may become disengaged and demotivated at work if they don't understand, or can't invest in, the "bigger picture."
```
Comment (Claiyc) [inline what-drives-us/what-drives-us.md:21]: People may become disengaged and demotivated at work if they don't understand or can't invest in the "bigger picture."

In english, no commas is in most cases better than too many commas.

File: what-drives-us/what-drives-us.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+But those who believe that they're working toward something larger and more important than themselves are often the most hardworking, productive and engaged.
```
Comment (Claiyc) [inline what-drives-us/what-drives-us.md:23]: "they are" instead of "they're"

**Review by Claiyc — CHANGES_REQUESTED** [review]

File: what-drives-us/what-drives-us.md (hunk 6 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md:21]: I think thats right the way it is?

**Comment by Claiyc** [comment]
Other than that, good work, very good summarization. I'm so proud of you guys! You are the best!

File: what-drives-us/what-drives-us.md (hunk 3 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md]: Fixed

File: what-drives-us/what-drives-us.md (hunk 4 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md]: fixed

**Review by jens1o — APPROVED** [review]

File: what-drives-us/what-drives-us.md (hunk 5 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md]: fixed

File: what-drives-us/what-drives-us.md (hunk 6 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md]: even if I disagree, i fixed it ;)

File: what-drives-us/what-drives-us.md (hunk 7 above)
Comment (XSnelliusX) [reply what-drives-us/what-drives-us.md]: fixed

**Review by Claiyc — APPROVED** [review]
Looks good to me!

File: what-drives-us/what-drives-us.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+## **Purpose**
+
+People may become disengaged and demotivated at work if they don't understand or can't invest in the "bigger picture."
+
+But those who believe that they are working toward something larger and more important than themselves are often the most hardworking, productive and engaged.
+
+---
+
+### Source
```
Comment (Illumin80) [inline what-drives-us/what-drives-us.md:27]: ### Sources

**Review by Illumin80 — CHANGES_REQUESTED** [review]

**Pull request merged**